components/title/basic/src/index.js

```diff
@@ -0,0 +1,55 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+export default function TitleBasic ({align, className, subtitle, title}) {
+  /**
+   * Builds the modifier className to apply to the component following the SUIT naming convention (sui-TitleBasic--modifierInCamelCase).
+   * @param alignProp {string} property value for Title alignment.
+   * @returns {string} SUIT formatted classname.
+   */
+  function _buildAlignClassName (alignProp) {
```

Comment:
It is not a private function but a function defined inside another function. IMO _ is not needed